auth/src/authorizers/default.ts

```diff
@@ -99,9 +101,7 @@ async function allowRegisteredDID(event: APIGatewayRequestAuthorizerEvent, callb
       if (error) {
         console.error(error.details)
       } else {
-        const createTableIfNotExists = false
-        const db = new DynamoDB(createTableIfNotExists)
-        const data = await db.addNonce(did, nonce)
+        const data = await dynamodbClient.addNonce(did, nonce)
```

Comment:
good catch!  if its long running tho and the handle closes will we need to reopen it?  Could we possibly test this branch in the keramik before merge to develop?

Reply:
I hope the sdk client would restart the connection 🙏 

The current problem is the lamba is running too long and we think it's opening too many connections.